src/components/controls.tsx

```diff
@@ -355,6 +414,24 @@ export class Controls extends BaseComponent<IProps, IState> {
     this.stores.setParticleSize(size);
   }
 
+  private changeVEI = (vei: number) => {
+    this.stores.setVEI(vei);
+  }
+
+  private getVEIDescription = (vei: number) => {
+    switch (vei) {
```

Comment:
Would recommend a map rather than a switch statement.

Reply:
Good call.  I'm going to move this and the VEI column height into a standardized object that we can pull values from and add future properties to if needed.